assets/js/base/components/cart-checkout/order-summary/order-summary-item.js

```diff
@@ -151,6 +159,20 @@ const OrderSummaryItem = ( { cartItem } ) => {
 					variation={ variation }
 				/>
 			</div>
+			<span className="screen-reader-text">
+				{ sprintf(
+					/* translators: %1$d is the number of items, %2$s is the total price including the currency symbol and %3$s is the item name. */
+					_n(
+						'Total price for %1$d %3$s item: %2$s',
```

Comment:
Not a big deal, but wondering if we should place variables in order, so `%2$s` is the name and `%3$s` the price?

Reply:
Done as well!